### ISSUE TITLE
nb-select with empty value

### ISSUE THREAD

**Issue by SashaSkywalker** [akveo/nebular#1235]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

**Current behavior:**
nb-select render empty string in label for option with empty value

**Expected behavior:**
nb-select should render option label

**Steps to reproduce:**
```
<nb-select>
  <nb-option value=''>first</nb-option>
  <nb-option value='2'>second</nb-option>
</nb-select>
```

if user selects first option then nothing will be rendered in label (bad)
if user selects second option then *second* will be rendered in label (good)

**Comment by rhajizadeh** [comment]
The same thing happens when with value equal to number 0 when we assign numbers to values.
for example:
```html
<nb-select>
  <nb-option [value]="0">
     first
   </nb-option>
   <nb-option [value]="1">
     test 2
  </nb-option>
</nb-select>
```
or: https://nb-select-zero-issue.stackblitz.io

**Comment by aefox** [comment]
PR for fix opened in https://github.com/akveo/nebular/pull/1282

**Comment by yggg** [comment]
Closing as fix (#1282) was merged
